android/src/main/java/com/oblador/keychain/KeychainModule.java

```diff
@@ -197,8 +266,8 @@ public void hasInternetCredentialsForServer(@NonNull String server, Promise prom
     }
 
     @ReactMethod
-    public void setInternetCredentialsForServer(@NonNull String server, String username, String password, String minimumSecurityLevel, ReadableMap unusedOptions, Promise promise) {
-        setGenericPasswordForOptions(server, username, password, minimumSecurityLevel, promise);
+    public void setInternetCredentialsForServer(@NonNull String server, String username, String password, ReadableMap options, String minimumSecurityLevel, Promise promise) {
```

Comment:
It also seems like the `options` and minimumSecurityLevel` params are taken in the wrong order here, compared to the JS code calling the function.